src/scripts/OSFramework/OSUI/Pattern/AnimatedLabel/AnimatedLabel.ts

```diff
@@ -60,7 +60,7 @@ namespace OSFramework.OSUI.Patterns.AnimatedLabel {
 
 		// Method that implements the toggle of the state of the input.
 		// It can either add or remove the class "active" of the input.
-		private _inputStateToggle(isFocus: boolean | undefined): void {
+		private _inputStateToggle(isFocus: boolean = false): void {
```

Comment:
You can remove the type here @jfranco09.
**isFocus = false**, since false is boolean, isFocus will be inferred as boolean.

Reply:
Just commit this change. Thank you for the feedback